dev/com.ibm.ws.jsp.2.3_fat/test-applications/TestInjection.war/src/com/ibm/ws/jsp23/fat/testinjection/tagHandler/JspCdiHitMeTag.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2015 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```

Comment:
This should be a date range `2015, 2020` please get this fixed in a new PR associated with the same issue. Thanks!